src/Components/ProjectPage/ProjectCreationForm.js

```diff
@@ -57,35 +57,64 @@ function Alert(props) {
 }
 
 export default function ProjectCreationForm(props) {
-  const [name, setName] = useState("");
-  const [description, setDescription] = useState("");
-  const [organization, setOrganization] = useState("");
-  const [organizationList, setOrganizationList] = useState([
-    "RMS",
-    "McDonald Australia",
-    "McDonald SG"
-  ]);
-  const [modelList, setModelList] = useState(["MCMC", "Brute Force"]);
-  const [defaultModel, setDefaultModel] = useState("");
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [organization, setOrganization] = useState('');
+  const [organizationList, setOrganizationList] = useState([]);
+  // const [modelList, setModelList] = useState(["MCMC", "Brute Force"]);
+  // const [defaultModel, setDefaultModel] = useState("");
   const [success, setSuccess] = useState(false);
+  const [fail, setFail] = useState(false);
+  const [empty, setEmpty] = useState(false);
 
   const onFormSubmission = e => {
     e.preventDefault();
     // ADD API CALL HERE
-    console.log(
-      "creating new project with name: ",
-      name,
-      ", description: ",
-      description,
-      ", organization: ",
-      organization,
-      ", model: ",
-      defaultModel
-    );
-    props.handleClose();
-    setSuccess(true);
+    if (name === '' || organization === '') {
+      setEmpty(true);
+    } else {
+      let newProject = {
+        title: name,
+        description: description,
+        company: organization
```

Comment:
Not working for normal user because user does not have access to /company/ endpoints
User should only be able to create project from his own company. So can remove this field for non-staff users and when sending post request to create project, you can retrieve company id by calling 
`store.getState().currentUser.company` after `import { store } from './redux/store'`
![image](https://user-images.githubusercontent.com/39354695/87268449-ff2a8200-c4fc-11ea-8226-d56ecd93a37f.png)